Sources/FaceLiveness/Utilities/CGImage+Convert.swift

```diff
@@ -0,0 +1,24 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import CoreGraphics
+import VideoToolbox
+
+extension CGImage {
+  static func convert(from cvPixelBuffer: CVPixelBuffer?) -> CGImage? {
```

Comment:
nit: Indentation seems to be 2 spaces instead of 4 in this file.

Reply:
fixed indentation